opentelemetry-stdout/src/common.rs

```diff
@@ -86,9 +86,21 @@ pub(crate) enum Value {
 
 impl PartialEq for Value {
     fn eq(&self, other: &Self) -> bool {
-        match (&self, &other) {
+        match (self, other) {
+            (Value::Bool(b), Value::Bool(ob)) => b.eq(ob),
+            (Value::Int(i), Value::Int(oi)) => i.eq(oi),
             (Value::Double(f), Value::Double(of)) => OrderedFloat(*f).eq(&OrderedFloat(*of)),
-            (non_double, other_non_double) => non_double.eq(other_non_double),
+            (Value::String(s), Value::String(os)) => s.eq(os),
+            (Value::Array(a), Value::Array(oa)) => a.eq(oa),
+            (Value::KeyValues(kv), Value::KeyValues(okv)) => kv.eq(okv),
+            (Value::BytesValue(b), Value::BytesValue(ob)) => b.eq(ob),
+            (Value::Bool(_), _) => false,
```

Comment:
Use a simpler wild card pattern instead?

```rust
_ => false,
```

Reply:
My concern was that if a new variant were added to `Value`, `PartialEq` would compile but have an incorrect result. With this version, there would be a compile error. But if you favor conciseness, I can change.

Reply:
Good point!